### DISCUSSION TITLE
Conflict with requirement numba 0.53.0 and Python 3.10

### DISCUSSION THREAD

**Discussion by sgoldstein527** [open-mmlab/mmdetection3d#2415] [Q&A] [answered]
I'm trying to install mmdetion3d from source and I run into the following issue:

#11 18.36 ERROR: Could not find a version that satisfies the requirement numba==0.53.0 (from mmdet3d)

It seems that numba 0.53.0 is not support on Python 3.10.  Is there a reason the numba version requirements could not be increased to 0.55.0?

Thanks!

**Comment by JingweiZhang12** [comment, marked as answer]
@sgoldstein527 We set numba==0.53.0 in order to be compatible with cuda9.0. If your use the cuda>10.0, just update the numba to the latest version.
